e2e-tests/spring-graphql/pom.xml

```diff
@@ -49,6 +49,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+            <version>2.5.0</version>
```

Comment:
recall [https://github.com/EMResearch/EvoMaster/blob/master/docs/for_developers.md#maven-dependency-version](). version declaration should be in root pom. if u are not sure how Maven work, we can chat about this next meeting

Reply:
Done